src/common/conditional_compilation/include/openvino/cc/opset/itt.hpp

```diff
@@ -0,0 +1,35 @@
+// Copyright (C) 2022 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#pragma once
+
+#include <openvino/cc/selective_build.h>
+
+#include <openvino/itt.hpp>
+
+OV_CC_DOMAINS(ov_opset);
```

Comment:
Why do we need this public header? I propose to move it to `src/core/src/itt.hpp`

Reply:
Fixed